### PR TITLE
Tutorial fix 2nd round

### PR DESCRIPTION
**Pull request by jbae11**
Second round of edits suggested by Kathryn Huff, Rober Flannagan, Baptiste Mouginot, and Paul Wilson at the dev call (09/14/2018)

### CODE REVIEW

**Comment by katyhuff** [comment]
ping... is this ready for review? I've forgotten about it. Sorry.

**Comment by jbae11** [comment]
yeah let me know what you think!

File: source/user/tutorial/add_proto.rst [hunk 1]
```diff
@@ -344,6 +355,12 @@ Let's take a look at the ``fresh-uox`` fuel recipe:
     :align: center
     :alt: Fuel recipe for fresh-uox
 
+|
```
Comment (katyhuff) [inline source/user/tutorial/add_proto.rst]: What's this about?

Comment (jbae11) [reply source/user/tutorial/add_proto.rst]: this is to provide the gap between the image and the next header so the change of topic is apparent.

image

Comment (katyhuff) [reply source/user/tutorial/add_proto.rst]: ah, thanks!

**Pull request merged**